posthog/api/capture.py

```diff
@@ -212,14 +214,17 @@ def get_event(request):
             ),
         )
 
-    ingestion_context, db_error, error_response = get_event_ingestion_context(request, data, token)
+    ingestion_context = None
+    send_events_to_dead_letter_queue = False
 
-    if error_response:
-        return error_response
+    if token not in settings.LIGHTWEIGHT_CAPTURE_ENDPOINT_ENABLED_TOKENS:
```

Comment:
Can we try the ingestion context, and on db connection error we then check ```if token not in settings.LIGHTWEIGHT_CAPTURE_ENDPOINT_ENABLED_TOKENS```

i.e. we always try to short circuit invalid events, but if we fail for a postgres unavailable reason then we simply push into and let Kafka consumer handle?

Reply:
hmm this is what I thought of doing at first, but actually I think we'd eventually want to get rid of this code, no? 

a bit of complexity to keep these in sync and think about code in two places, plus getting rid of it unlocks spinning the capture service out in the future if we ever want to do that

Reply:
Yeah a little annoying to keep in sync, happy to go with this.

In a world where the consuming part and the HTTP handler were able to utilise the same code, either directly or via some other method (e.g. some http endpoint the provided this information), we wouldn't have a sync issue.